src/Frontend/Page/Topic.hs

```diff
@@ -3,6 +3,7 @@
 {-# LANGUAGE FlexibleInstances   #-}
 {-# LANGUAGE LambdaCase          #-}
 {-# LANGUAGE OverloadedStrings   #-}
+{-# LANGUAGE RecordWildCards     #-}
```

Comment:
Fair enough, that's a genuine trade-off you are talking about and I respect your preference. Fixed.